test/Microsoft.DotNet.ApiCompatibility.Tests/Logging/SuppressionEngineTests.cs

```diff
@@ -60,8 +60,9 @@ public void SuppressionEngine_WriteSuppressionsToFile_SuppressionsRoundTrip()
             suppressionEngine.LoadSuppressions("NonExistentFile.xml");
 
             string filePath = Path.Combine(Path.GetTempPath(), Path.GetTempFileName(), "DummyFile.xml");
-            IReadOnlyCollection<Suppression> writtenSuppressions = suppressionEngine.WriteSuppressionsToFile(filePath, preserveUnnecessarySuppressions: true);
+            var (suppressionFileUpdated, writtenSuppressions) = suppressionEngine.WriteSuppressionsToFile(filePath, preserveUnnecessarySuppressions: true);
 
+            Assert.True(suppressionFileUpdated);
```

Comment:
Should we add a test case for the scenario where we had baselines and they are no longer needed -- ensure that the file is emptied and true is returned?

Reply:
I just noticed that we already have one that validates that: `SuppressionEngine_WriteSuppressionsToFile_ReturnsEmptyWithAllUnnecessarySuppressions`